VAMobile/src/store/slices/appointmentsSlice.ts

```diff
@@ -449,6 +451,11 @@ const appointmentsSlice = createSlice({
       state.loading = false
       state.preloadComplete = true
 
+      if(state.firstTimeLoading === true && upcoming && upcoming.meta) {
+        logAnalyticsEvent(Events.vama_hs_appts_count(upcoming?.meta?.upcomingAppointmentsCount))
```

Comment:
```suggestion
        await logAnalyticsEvent(Events.vama_hs_appts_count(upcoming.meta.upcomingAppointmentsCount))
```

But I would also move this to the `prefetchAppointments` function, since reducers are supposed to be pure functions. Probably right before this [line](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/9d7a5c29787042f886d36c2e242531554f502566/VAMobile/src/store/slices/appointmentsSlice.ts#L289)